charts/aws-efs-csi-driver/templates/csidriver.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: storage.k8s.io/v1beta1
+apiVersion: storage.k8s.io/v1
```

Comment:
This would be better handled like this and then you wouldn't need to increase the minimum kube version.
https://github.com/kubernetes-sigs/aws-ebs-csi-driver/pull/856/files#diff-e167822724b087c64ed35c6f75c8af5d054a981e00b59488145796c69c6c611fR1

`apiVersion: {{ ternary "storage.k8s.io/v1" "storage.k8s.io/v1beta1" (semverCompare ">=1.18.0-0" .Capabilities.KubeVersion.Version) }}`

Reply:
Fixed in b1c989de876b5cc719153f51dbe3a831a0a731ed